pkg/controller/installation/prometheusRules.go

```diff
@@ -62,6 +62,23 @@ func (r *ReconcileInstallation) newAlertsReconciler(logger *logrus.Entry, instal
 					},
 				},
 			},
+			{
+				AlertName: "rhmi-upgrade-alerts",
+				Namespace: "openshift-monitoring",
+				GroupName: "rhmi-upgrade.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHMIUpgradeTooLong",
+						Annotations: map[string]string{
+							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
```

Comment:
Is there a specific section in this SOP for this alert? It might get rejected in the near future for not meeting SRE requirements for not having an actionable SOP.

Reply:
I created a PR to include the alert in the SOP: 
https://github.com/RHCloudServices/integreatly-help/pull/642